tsc-2017.yaml

```diff
@@ -0,0 +1,245 @@
+name: TSC
+CC1.1:
```

Comment:
Should these names have spaces, e.g. ``CC 1.1``?

Attempted to check, but looks like the TSP's are behind a paywall.

Reply:
Just checked: they seem to be pervasively referenced without the space (definitely CC1.5, PI1.6, etc) in both the source material as well as the auditor documents I have reviewed. Will stick with this format.